ilastik/workflows/carving/carvingGui.py

```diff
@@ -259,7 +256,7 @@ def generateObjectName():
 
         def validate():
             name = dialog.lineEdit.text()
```

Comment:
I think it's fine to unconditionally change widget properties here, and do this even if `isDisabled` is false.
```python
valid = name not in existing_names
# Could also remove double negation here and rename the field to isEnabled.
dialog.isDisabled = not valid
dialog.Ok.setEnabled(valid)
dialog.warning.setVisible(not valid)
```

Reply:
We probably should `name = name.strip()` and treat an empty name as invalid.